boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java

```diff
@@ -173,6 +174,108 @@ public void testPodAuthors() throws Exception {
         assertEquals(podAuthors, openAPIDevs);
     }
 
+    @Test
+    public void testFromModel() {
```

Comment:
Would be nice to include a schema that meets the criteria for the above fix (free form object without additional properties).

Reply:
I've added schemas that meet the criteria.